### PR TITLE
Debugging and Run/Debug Configurations

### PR DESCRIPTION
**Pull request by arnautolea**

### CODE REVIEW

File: JT/07_bedugging_and_run_debug_configuration/ IDE_Debugger/How_to_debug.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  * [Step through the program](#step-through-the-program)
+  * [Stop the debugger session and rerun the program](#stop-the-debugger-session-and-rerun-the-program)
+  * [Action shortcuts](#action-shortcuts)
+  * [Resources](#resources)
+<!-- TOC -->
+
+## Overview
+***
+
+Debugger makes application debugging much easier. Using debugger, we can **stop** the execution of program at a certain point, **inspect variables**, **evaluate** and some other things. IntelliJ provides inbuilt Java debugger.
```
Comment (tascaV) [inline JT/07_bedugging_and_run_debug_configuration/ IDE_Debugger/How_to_debug.md:22]: stop -> pause

**Review by tascaV — APPROVED** [review]
Great work @arnautolea !!! A trivial comment, otherwise good to be merged from my point of view.

**Comment by endalexei** [comment]
Good work Olga, I would add some information about evaluate expression and modify data in run.

File: JT/07_bedugging_and_run_debug_configuration/run_debug_configurations/Run_Debug_configurations.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+If an argument includes spaces, enclose the spaces or the argument that contains spaces in double quotes, for example, some" "arg or "some arg".
+
+If an argument includes double quotes (as part of the argument), escape the double quotes using backslashes, for example, \"quoted_value\".
+
+
+## Resources
+***
+
+- [Run/debug configurations](https://www.jetbrains.com/help/idea/run-debug-configuration.html#config-folders)
```
Comment (Vehkiya) [inline JT/07_bedugging_and_run_debug_configuration/run_debug_configurations/Run_Debug_configurations.md:151]: Let's add 1 more resource that incldues remote debugging
https://www.jetbrains.com/help/idea/tutorial-remote-debug.html#create-run-configurations

Comment (arnautolea) [reply JT/07_bedugging_and_run_debug_configuration/run_debug_configurations/Run_Debug_configurations.md:151]: done

File: JT/07_bedugging_and_run_debug_configuration/ IDE_Debugger/How_to_debug.md (hunk 1 above)
Comment (arnautolea) [reply JT/07_bedugging_and_run_debug_configuration/ IDE_Debugger/How_to_debug.md:22]: ok, done

**Pull request merged**